.babelrc

```diff
@@ -1,3 +1,3 @@
 {
-  "presets": ["node6"]
+  "presets": ["es2015"]
```

Comment:
I would recommend using this preset instead: https://github.com/gabmontes/babel-preset-latest-minimal